backend/app/services/memory/manager.py

```
@@ -0,0 +1,377 @@
+                model_config_override=model_config_override,
+            )
+
+            logger.info(
+                "Retrieved %d memories for user %d, task %d, team %d",
+                len(result.results),
+                user_id,
+                task_id,
+                team_id,
+            )
+
+            return result.results
```

Ruff (0.14.11)

Consider moving this statement to an `else` block

(TRY300)